public/docs/_examples/webpack/ts/tsconfig.1.json

```diff
@@ -6,8 +6,9 @@
     "sourceMap": true,
     "emitDecoratorMetadata": true,
     "experimentalDecorators": true,
+    "lib": ["es2015", "dom"],
     "removeComments": false,
```

Comment:
Would you please remove this `removeComments` setting too. That is the default setting and we don't want it here. I thought I'd removed it everywhere but I guess I missed this one.

Then I can merge your PR.